### ISSUE TITLE
mysql and mongo together in the same project

### ISSUE THREAD

**Issue by ceresaconsultoria** [jenssegers/laravel-mongodb#1841]
I was able to configure it this way, where mongo is the default connection and mysql is the secondary connection

```
       $container['database'] = function ($c) {
        $settings = $c->get('settings');
        
        $capsule = new Capsule;

        $capsule->getDatabaseManager()->extend("mongodb", function ($config) {
            return new Jenssegers\Mongodb\Connection($config);
        });
        
        $capsule->addConnection($settings['db']["mysql"], 'write');

        $capsule->addConnection($settings['db']["mongodb"]);
        
        $capsule->bootEloquent();
        
        $capsule->setAsGlobal();
        
        return $capsule;
    };
```
Inside my model I need to set mysql connection to work

```
<?php

namespace Tooview\Core;

use Illuminate\Database\Eloquent\Model as Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class ModelWrite extends Model{
    use SoftDeletes;
    
    protected $connection = 'write';
    protected $dates = ['deleted_at'];
}
```
Is there another way to make the connection?

**Comment by Smolevich** [comment]
I think that no

**Comment by Giacomo92** [comment]
Hi @ceresaconsultoria, 
you can also create 2 Model:

-  **MysqlBaseModel** that extends the default Laravel model `Illuminate\Database\Eloquent\Model` and has `protected $connection = 'mysql';` value set as the name of the Mysql connection
- **MongoDbBaseModel**: that extends the MongoDB model `Jenssegers\Mongodb\Eloquent\Model` and has `protected $connection = 'mongodb';` value set as the name of the MongoDb connection

Now you can choose what model extend in order to use mongoDb or Mysql connection. 
The default connection is useful if you have only one database in the project, so you can avoid declaring`protected $connection` in your models.

**Comment by ceresaconsultoria** [comment]
Hi,

Testimonials carousel bug

[image: WhatsApp Image 2020-01-27 at 11.40.34.jpeg]

[image: WhatsApp Image 2020-01-27 at 11.40.34 (1).jpeg]

When you set an event as passive, it is already preventing default behavior
ai n it makes sense to put a preventDefault (which was the possible failure
of those who import the library), then the passive true comes and bugs the
part that was defined to have carousel.
  The scroll response only happens when the gesture ends, in case the mouse
passes there and drag.
I simulated the code here, in a test, and I think I managed to find the
way, finishing
the possible solution to fix the issue.
However, it is good to get in touch with the people of the subject, and say
that it came buggy.
It’s a theme that comes with akele page builders, so I really
I am changing little css in the root, the necessary one. These carousel
areas
are affected by it.

[image: image.png]

--



Giacomo Fabbian <notifications@github.com> escreveu no dia sexta,
10/01/2020 à(s) 19:28:
